packages/scheduler/src/Scheduler.js

```diff
@@ -144,14 +144,8 @@ function flushTask(task, currentTime) {
   // with the same priority and expiration as the just-finished callback.
   if (typeof continuationCallback === 'function') {
     var expirationTime = task.expirationTime;
-    var continuationTask = {
-      callback: continuationCallback,
-      priorityLevel: task.priorityLevel,
-      startTime: task.startTime,
-      expirationTime,
-      next: null,
-      previous: null,
-    };
+    var continuationTask = task;
```

Comment:
is it desirable to mutate `task.callback`, `task.next` and `task.previous`? if not, the `task` object needs to be copied, not simply assigned.

Reply:
It's on purpose.